vochain/scrutinizer/process.go

```diff
@@ -33,6 +40,15 @@ func (s *Scrutinizer) ProcessList(entityID []byte, namespace uint32,
 			return nil, fmt.Errorf("processList: status %s is unknown", status)
 		}
 	}
+	searchMatchFunc := func(r *badgerhold.RecordAccess) (bool, error) {
+		if searchTerm == "" {
+			return true, nil
+		}
+		if strings.Contains(hex.EncodeToString(r.Field().(types.HexBytes)), searchTerm) {
+			return true, nil
+		}
+		return false, nil
+	}
```

Comment:
I know this func is small, but it's tricky and we have it in two places already - I'd make it a global func and reuse it.

I'd also add a TODO above the hex encoding stuff, because we can likely do better in the future, as we discussed before. For example, doing "raw bytes" search if the search term is complete hex.